src/utils/multiSig.ts

```diff
@@ -1,13 +1,15 @@
-import { UserRole } from '~constants/permissions.ts';
+import { type ColonyRole } from '@colony/colony-js';
+
+import { PERMISSIONS_NEEDED_FOR_ACTION } from '~constants/actions.ts';
 import { ColonyActionType } from '~gql';
 
-export const getMultiSigRequiredRole = (
+export const getRolesNeededForMultiSigAction = (
   actionType: ColonyActionType,
-): UserRole => {
+): ColonyRole[] | undefined => {
   switch (actionType) {
     case ColonyActionType.MintTokensMultisig:
```

Comment:
Just a note that we forgot to add the unlock token action type here, so we aren't able to create that one, but we can fix this after it's merged since this PR isn't at fault